Nimble/Matchers/SatisfyAnyOf.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+public func satisfyAnyOf<T,U where U: Matcher, U.ValueType == T>(matchers: U...) -> NonNilMatcherFunc<T> {
```

Comment:
Please add some documentation comments for the public matcher functions. It helps when testers are typing / jump to definition.